symengine/lib/symengine.pxd

```diff
@@ -769,6 +769,18 @@ cdef extern from "<symengine/matrix.h>" namespace "SymEngine":
         DenseMatrix(unsigned i, unsigned j) nogil
         DenseMatrix(unsigned i, unsigned j, const vec_basic &v) nogil
         void resize(unsigned i, unsigned j) nogil
+        void row_join(const DenseMatrix &B) nogil
+        void col_join(const DenseMatrix &B) nogil
+        void row_insert(const DenseMatrix &B, unsigned pos) nogil
+        void col_insert(const DenseMatrix &B, unsigned pos) nogil
+        void row_del(unsigned k) nogil
+        void col_del(unsigned k) nogil
+        void row_exchange_dense(DenseMatrix &A, unsigned i, unsigned j) nogil
+        void row_mul_scalar_dense(DenseMatrix &A, unsigned i, RCP[const Basic] &c) nogil
+        void row_add_row_dense(DenseMatrix &A, unsigned i, unsigned j, RCP[const Basic] &c) nogil
+        void column_exchange_dense(DenseMatrix &A, unsigned i, unsigned j) nogil
+        void dot(const DenseMatrix &A, const DenseMatrix &B, DenseMatrix &C) nogil
+        void cross(const DenseMatrix &A, const DenseMatrix &B, DenseMatrix &C) nogil
```

Comment:
They are not declared in class `DenseMatrix`. They should be declared outside the class.